docs/import/add_disaster.js

```diff
@@ -359,14 +366,88 @@ function populateStateAssetPickers() {
       for (const asset of assets) {
         stateAssets.set(asset[0], asset[1]);
       }
-      createAssetPickers(states);
+      createAssetPickers(states, 'asset-pickers');
+      initializeScoreSelectors(states);
     });
   }
 
   // TODO: display more disaster info including current layers etc.
   return assetPickersDone;
 }
 
+/**
+ * Initializes the select interface for score assets.
+ * @param {Array<string>} states array of state (abbreviations)
+ */
+function initializeScoreSelectors(states) {
+  const headerRow = $('#score-asset-header-row');
+  const tableBody = $('#asset-selection-table-body');
+  tableBody.empty();
+  headerRow.empty();
+
+  // Initialize headers.
+  headerRow.append(createTd().html('Score Assets'));
```

Comment:
Where will picking the damage asset happen? and maybe we should rephrase Score Assets since income and svi aren't actually included in the score. But happy to leave the latter as a thought and see if it confuses Alex or not.